features/step_definitions/user_steps.rb

```diff
@@ -45,6 +45,11 @@
 
 Given /^I am logged in as "([^\"]*)" with password "([^\"]*)"$/ do |login, password|
   step("I am logged out")
+  step(%{I am logged in as "#{login}" with password "#{password}" no logout step})
+  assert UserSession.find
+end
+
+Given /^I am logged in as "([^\"]*)" with password "([^\"]*)" no logout step(?:( with things hidden))?$/ do |login, password, hidden|
```

Comment:
It was trying to reduce the number of steps in the feature, I wanted to log in and have the various preferences to do with hiding warnings and additional tags set.